blog/_posts/2020-07-15-Tetiana-Dadakova.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: "Future blogpost about MABE"
+date: 2020-07-15
+author: Tetiana
+---
+
+# Genome change logging using range map  
+
+For more information about MABE (Modular Agent-Based Evolution platform) see [this introduction](https://szendejo.github.io/waves/blog/Team-MABE.html).
+
+## Genome Class
+
+### Naïve Implementation
+
+The naïve implementation of the genome is using a vector data structure from the standard library. The overwrite, insert and remove operations are, therefore, implemented using the standard library algorithms, specifically:
```

Comment:
I would specifically mention that using a vector data structure is a way of representing a genome. So maybe something like: "The naïve way to represent a genome with code is ..." 

I would also include a brief introductory sentence as to what the overall goal of the genome class is